brainstormforce/ultimate-addons-for-gutenberg
# UAG-349 Fix: UAG Styling not applying on preview of page.

### Description

- Card - https://ua-gutenberg.atlassian.net/jira/software/projects/UAG/boards/2?selectedIssue=UAG-349

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a page drag and drop UAG block.
- Add a styling to the block and preview the page
- It should display all style on preview page.
- Update the page and visits the front-page and check preview and front-page style.

### Checklist:
